net/mdio-tools/Makefile

```diff
@@ -5,9 +5,9 @@ PKG_RELEASE:=1
 
 PKG_SOURCE_URL:=https://github.com/wkz/mdio-tools
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_DATE:=2021-07-19
-PKG_SOURCE_VERSION:=65f6898f25b00deebe1c38811e893d9dcd4d7ef3
-PKG_MIRROR_HASH:=9db854037b3e8596ccd71567a45aa60fd3cac88b01c9197d4dd9cf3f32589cb2
+PKG_SOURCE_DATE:=2021-09-19
```

Comment:
No need when you have a PKG_SOURCE_VERSION like this.

Reply:
So, drop PKG_SOURCE_DATE totally ?

Reply:
Yeah.